tests/std/tests/P0528R3_cmpxchg_pad/test.cpp

```diff
@@ -29,6 +33,8 @@ struct X3 {
     char y : 1;
     short z;
 
+    void operator&() const = delete;

```

Comment:
Should `X4` below receive `void operator&() const = delete;` too? It's the only one missing this now.